File: src/main/javascript/leetcode/P1480_Running_Sum_of_1d_Array.js
var runningSum = function(nums) {
    var result = [];
    for (var i = 0; i < nums.length; i++) {
        var sum = 0;
        if (i > 0) {
            sum = result[i - 1];
        }
        sum += nums[i];
        result.push(sum);
    }
    return result
};

console.log(runningSum([1,2,3,4]));

File: src/main/java/leetcode/P1480_Running_Sum_of_1d_Array.java
package leetcode;

public class P1480_Running_Sum_of_1d_Array {
    public int[] runningSum(int[] nums) {
        int[] result = new int[nums.length];
        for (int i = 0; i < result.length; i++) {
            int sum = 0;
            if (i != 0) {
                sum += result[i - 1];
            }
            sum += nums[i];
            result[i] = sum;
        }
        return result;
    }
}
